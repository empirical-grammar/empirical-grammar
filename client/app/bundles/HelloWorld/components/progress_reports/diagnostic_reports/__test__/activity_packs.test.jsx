import React from 'react';
import { shallow } from 'enzyme';

import ActivityPacks from '../activity_packs.jsx'

import processEnvMock from '../../../../../../../__mocks__/processEnvMock.js';
window.process = processEnvMock;

describe('ActivityPacks component', () => {

  it('should render', () => {
    document.body.innerHTML =
  '<div>' +
  '  <div class="diagnostic-tab" />' +
  '  <button class="activity-analysis-tab" />' +
  '</div>';

    const wrapper = shallow(
        <ActivityPacks />
    );
    expect(wrapper).toMatchSnapshot();
  });

});
