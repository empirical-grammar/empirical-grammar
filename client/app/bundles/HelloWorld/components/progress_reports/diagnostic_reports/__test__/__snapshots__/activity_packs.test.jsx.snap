// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ActivityPacks component should render 1`] = `
<div
  className="container manage-units"
>
  <loading_indicator />
</div>
`;
